patterns/footer-portfolio.php

```diff
@@ -58,7 +58,7 @@
 
 <!-- wp:group {"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between"}} -->
 <div class="wp-block-group"><!-- wp:paragraph {"fontSize":"small"} -->
-<p class="has-small-font-size">© 2024 Twenty Twenty Four</p>
+<p class="has-small-font-size"><?php echo esc_html_x( '© 2024 Twenty Twenty Four', '', 'twentytwentyfour' ); ?></p>
 <!-- /wp:paragraph -->
 
 <!-- wp:paragraph {"fontSize":"small"} -->
```

Comment:
You also follow the footer.php, and enable the translate 
`<p class="has-small-font-size">Designed with <a rel="nofollow" href="https://wordpress.org">WordPress</a></p>`

Reply:
Please check these lines: https://github.com/WordPress/twentytwentyfour/blob/trunk/patterns/footer.php#L16-L24